deploy/ui/form.json

```diff
@@ -841,6 +841,23 @@
                   "type": "text",
                   "description": "The secret name within the Java keystore used for https encryption. Generated if empty"
                 },
+                {
+                  "label": "JBPM Cluster",
```

Comment:
As we have jbpmCluster object in KieApp, it would be better to have this configuration group in UI as well, and visible only in case jbpmCluster is enabled. (E.g. similar like we have immutable server configuration)

Reply:
ok